corehq/apps/case_search/tests/test_filter_dsl.py

```diff
@@ -30,20 +30,18 @@
 from corehq.apps.es.cases import case_name, is_closed
 from corehq.apps.es.tests.utils import ElasticTestMixin, es_test
 from corehq.form_processor.tests.utils import FormProcessorTestUtils
-from corehq.util.test_utils import flag_enabled
 
 
 @es_test
 class TestFilterDsl(ElasticTestMixin, SimpleTestCase):
 
     def test_simple_filter(self):
-        parsed = parse_xpath("name = 'farid'")
-        expected_filter = case_property_query('name', 'farid')
```

Comment:
Just curious why did "name" need to be changed here?

Reply:
name is a special case property, so the optimizations now search against `case.name` instead of `{'case_properties.key': 'name', 'case_properties.value': 'SEARCH_STRING'}`.  The point of this test was to test a normal case property query, so I changed it to something else.